ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -982,6 +982,16 @@ FCU_DFSS_CHWZC_RTM_ZHM:
     - CHWZC
     - RTM
     - ZHM
+
+FCU_CHWZTC_HWZTC_DFVSC_RTM:
```

Comment:
please add DFSS and check that this type isnt covered by another, existing type.

I see this type: `FCU_DFSS_DFVSC_ZTC_CHWZTC_HWZTC`; does that not cover it? I think `RTM` is covered by the other types.

Reply:
I added RTM for the return_air_temperature_sensor. Actually, FCU_DFSS_DFVSC_ZTC_CHWZTC_HWZTC do not cover it and neither the other existing type. 
I can ever add the return_air_temperature_sensor as an optionnal point in one of those type (ZTC, HWZTC, CHWZTC) or omit the return_air_temperature_sensor of my modelisation as it is not included in the regulation (this point is just for information).

Reply:
What field is missing, just the `return_air_temperature_sensor`? If so, please extend with `FCU_DFSS_DFVSC_ZTC_CHWZTC_HWZTC_RTM`. I would like `DFSS` and `ZTC` included explicitly.

Reply:
Yes exactly, just return_air_temperature_sensor. What do you mean by extend with FCU_DFSS_DFVSC_ZTC_CHWZTC_HWZTC_RTM ? You mean create a non standard type that implement FCU_DFSS_DFVSC_ZTC_CHWZTC_HWZTC_RTM and use return_air_temperature_sensor ?

Reply:
sorry, i mean please turn this type into `FCU_DFSS_DFVSC_ZTC_CHWZTC_HWZTC_RTM` by adding the necessary implemented types and changes the name.